op-chain-ops/genesis/genesis.go

```diff
@@ -40,6 +40,10 @@ func NewL2Genesis(config *DeployConfig, l1StartHeader *types.Header) (*core.Gene
 	}
 
 	l1StartTime := l1StartHeader.Time
+	var soulGasTokenBlock *uint64
+	if config.UseSoulGasToken {
+		soulGasTokenBlock = u64ptr(config.SoulGasTokenBlock)
```

Comment:
What is the benefit of u64ptr vs &?

Reply:
It's the same, but more consistent with the existing code.